domain/infosync/info.go

```diff
@@ -358,8 +430,34 @@ func (is *InfoSyncer) newSessionAndStoreServerInfo(ctx context.Context, retryCnt
 		err := is.storeServerInfo(ctx)
 		return errors.Trace(err)
 	})
-	err = is.storeServerInfo(ctx)
-	return err
+	return is.storeServerInfo(ctx)
+}
+
+// newTopologySessionAndStoreServerInfo creates a new etcd session and stores server info to etcd.
+func (is *InfoSyncer) newTopologySessionAndStoreServerInfo(ctx context.Context, retryCnt int) error {
+	if is.etcdCli == nil {
+		return nil
+	}
+	logPrefix := fmt.Sprintf("[topology-syncer] %s", is.serverInfoPath)
```

Comment:
The `logPrefix` of topology session should not use the `is.serverInfoPath`, which may cause confusion.